docker/node/Dockerfile

```diff
@@ -4,7 +4,9 @@ FROM node:8-slim
 
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get --no-install-recommends install -y \
+    apt-utils \
+    ca-certificates \
```

Comment:
I get this message: 

```
ca-certificates is already the newest version (20161130+nmu1+deb9u1).
```

It is already installed in the base image, so we don't need to install it again. Please remove this line.

Reply:
done , may be already in the image . I added because without this wget , curl or git , gives certificate error